### PR TITLE
Chore/bootstrap4

### PR DESCRIPTION
**Pull request by jnizet**
This moves the blog to bootstrap 4, and the latest font-awesome (using a CDN, but we could use our own file) for the icons.

It's split in 3 commits:
- the first one moves to bootstrap 4 and adjusts some styles for simplification, spacing, alignment and responsiveness, but stays close to the look we have now
- the second one removes the ugly background color and replaces it by a border and a box-shadow, that I like better
- the third one uses a serif font for the body, to improve the reading experience
- the fourth one removes the border, box-shadow, and makes some other small adjustments that we agreed on with Cédric.

Please tell me what you think.

### CODE REVIEW

**Comment by cexbrayat** [comment]
I added two small commits:
- one to add a bit more margin around subtitles in text
- one to use regular instead of solid icon when posible, and to have them in the same color as the text there are next to (orange for bookmarks and comments)

capture d ecran 2018-08-30 a 11 59 02

File: _includes/themes/twitter/default.html [hunk 1]
```diff
@@ -81,15 +79,15 @@
...
+            <span class="far fa-envelope"></span>
             Contact
           </a>
         </li>
       </ul>
       <ul class="navbar-nav">
         <li>
           <a class="nav-link" href="/atom.xml" title="Atom feed">
-            <span class="fas fa-rss-square"></span>
+            <span class="far fa-rss-square"></span>
```
Comment (jnizet) [inline _includes/themes/twitter/default.html]: This icon doesn't exist in regular mode.

File: assets/themes/twitter/css/style.css [hunk 2]
```diff
@@ -20,12 +20,15 @@ a:hover, a.page-link:hover {
   color: #ea9960;
 }
 
-/* The white background content wrapper */
 .content {
   padding: 20px;
   margin: 0 -20px; /* negative indent the amount of the padding to maintain the grid system */
 }
 
+.fa-bookmark, .fa-comment-alt {
```
Comment (jnizet) [inline assets/themes/twitter/css/style.css]: Why not simply include them inside the link, rather than leaving them outside and use the same color?

**Comment by clacote** [comment]
Looks good to me.

Some details are bothering me:

- the line height for the code blocks are too high, rendering too much margin between lines of code:
code

- the icons on the navbar look inconsistent. I don't know the Font Awesome terminology for this (regular vs solid?),but they are mostly "empty" (small border on transparent background), except two of them which look "plain" (solid background), making them accentuated, whereas they're not supposed to be:
  navbar icons

- [subjective] Don't you think that something  is missing to make the main title (`<h1>`) standout more? Bolder font?

- [subjective] Orange for the links looks out of the Ninja Squad visual guide, if such a think existed. But as I suck with colors, I don't have a better suggestion. Orange is nice to have a great contrast with regular text, though.

**Comment by cexbrayat** [comment]
- I agree for the line height of the code
- sadly all icons are not available in regular forms in the free version. We can either let it like this, make them all solid to be consistent, or buy the pro version to switch all to regular. WDYT?
- Orange links were already used, but we can switch. What would you prefer?

**Comment by cexbrayat** [comment]
I pushed 3 commits for Cyril's feedback.

**Comment by acrepet** [comment]
I know I'm late! Great!!! 
Just to say one nitpicking :
I don't know why (I didn't see any change about that in the code) but there is a lot of space between comments and title of the post, it's a little bit weird

**Comment by cexbrayat** [comment]
@acrepet I tried to fix it in #274 , tell me what you think when you have the occasion

**Comment by acrepet** [comment]
I've tested, perfect 👍

**Pull request merged**